services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/JunctionTest.java

```diff
@@ -0,0 +1,56 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class JunctionTest extends TestUtils {
```

Comment:
Maybe we should also check the default object? That "name" is null?

Reply:
Do you mean that `Junction` object is not null even though `junction.name` is null?

Reply:
Yes. You say in the docs that name is optional.

Reply:
Done